doc/whats_new/v1.4.rst

```diff
@@ -300,6 +300,12 @@ Changelog
   which can be used to check whether a given set of parameters would be consumed.
   :pr:`26831` by `Adrin Jalali`_.
 
+- |Fix| :func:`sklearn.utils.check_array` should accept both matrix and array from
+  the sparse SciPy module. The current implementation would fail if `copy=True` by
```

Comment:
At the time this is read by a user, "current" will not mean the same thing.

```suggestion
  the sparse SciPy module. The previous implementation would fail if `copy=True` by
```